src/main/resources/templates/fragments/internal-server-error.html

```diff
@@ -1,6 +1,7 @@
 <div th:if="${session.get('errorModel') != null and session.get('errorModel').getError().equals('unknown')}">
     <h1>Ein unerwarteter Fehler ist aufgetreten</h1>
     <p th:text="${session.get('errorModel').getErrorMessage()}"></p>
-    <p>Kontaktieren Sie doch bitte <a class="link"
-            href="mailto: helpdesk@unilu.ch">helpdesk@unilu.ch</a></p>
+    <p>Kontaktieren Sie doch bitte
```

Comment:
Per Du oder per Sie? Entweder neutral formulieren oder per Du (wie im not-found-error.html).